test/common/json/json_sanitizer_test_util.h

```diff
@@ -23,6 +23,32 @@ bool utf8Equivalent(absl::string_view a, absl::string_view b, std::string& errms
     EXPECT_TRUE(TestUtil::utf8Equivalent(a, b, errmsg)) << context << "\n" << errmsg;              \
   }
 
+/**
+ * Determines whether `sanitized`is equivalent to `original`. `original` may be
```

Comment:
```suggestion
 * Determines whether `sanitized` is equivalent to `original`. `original` may be
```

minor spelling error.

Reply:
@jmarantz